common/flatpak-utils.c

```diff
@@ -7532,23 +7532,23 @@ gboolean
 flatpak_dconf_path_is_similar (const char *path1,
                                const char *path2)
 {
-  int i, i1, i2;
+  int i1, i2;
   int num_components = -1;
 
-  for (i = 0; path1[i] != '\0'; i++)
+  for (i1 = i2 = 0; path1[i1] != '\0'; i1++, i2++)
     {
-      if (path2[i] == '\0')
+      if (path2[i2] == '\0')
         break;
 
-      if (tolower (path1[i]) == tolower (path2[i]))
+      if (tolower (path1[i1]) == tolower (path2[i2]))
         {
-          if (path1[i] == '/')
+          if (path1[i2] == '/')
```

Comment:
I know they are the same here, but should this not be i1?

Reply:
It definitely should be.